components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__genes.xml

```diff
@@ -137,10 +137,14 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
-          element.insert({after: infoTool});
-        else
+        if (isSuggestSelected) {
+          if ($('inline')) {
```

Comment:
Also the infoTool and triangle elements are not conditionally generated even though they are conditionally placed on the screen.